lib/orcid/add_works.rb

```diff
@@ -21,8 +21,8 @@ def add_all(orcid_users)
     def add_for_orcid_user(orcid_user)
       return 0 unless orcid_user.update?
 
-      author = Author.find_by(sunetid: orcid_user.sunetid)
-      return 0 if author.nil? || author.cap_visibility != 'public'
+      author = Author.find_by(sunetid: orcid_user.sunetid, active_in_cap: true, cap_visibility: 'public')
+      return 0 if author.nil?
 
```

Comment:
look for both "active" and "public" when pushing to ORCID.  Slight syntax change to make this clearer and done via query only.